packages/helm/symphony/templates/symphony-core/symphonyk8s.yaml

```diff
@@ -2705,6 +2705,26 @@ webhooks:
     resources:
     - solutions
   sideEffects: None
+- admissionReviewVersions:
```

Comment:
Did you try mage helmTemplate to rebuild this file?

Reply:
Yes. Any concerns?

Reply:
No concerns. We need to make sure symphonyk8s.yaml can be auto-generated.